cirq-core/cirq/experiments/two_qubit_xeb.py

```diff
@@ -53,6 +53,46 @@ def _manhattan_distance(qubit1: 'cirq.GridQubit', qubit2: 'cirq.GridQubit') -> i
     return abs(qubit1.row - qubit2.row) + abs(qubit1.col - qubit2.col)
 
 
+def _qubits_and_pairs(
```

Comment:
this function shouldn't be local.. we will need it in other files
```suggestion
def qubits_and_pairs(
```